shared/constants/permissions.ts

```diff
@@ -18,11 +18,21 @@ export const RestrictedMethods = Object.freeze({
 } as const);
 
 ///: BEGIN:ONLY_INCLUDE_IN(flask)
+const isMain = process.env.METAMASK_BUILD_TYPE === 'main';
```

Comment:
Could we define these in a shared location and export? We need one in the MetaMask controller for `isFlask` too.